src/layouts/Layout.astro

```diff
@@ -130,8 +130,6 @@ const bannerOffset = bannerOffsetByPosition[siteConfig.banner.position || 'cente
 
 		<link rel="alternate" type="application/rss+xml" title={profileConfig.name} href={`${Astro.site}rss.xml`}/>
 
-		<style define:vars={{ configHue }}></style>  <!-- defines global css variables. This will be applied to <html> <body> and some other elements idk why -->
```

Comment:
Here too. I suggest keeping this style element to store the CSS variables (moving `bannerOffset` here as well).

Reply:
removing `opacity: 0` from ImageWrapper fixed the issue. But i'm not very into the project so i'm probably wrong

Reply:
resolve